### ISSUE TITLE
The User Account wasn't Found or is Protected

### ISSUE THREAD

**Issue by zherebilovgg** [mahrtayyab/tweety#212]
Hi,
When I use  `app.get_tweets` method I sometimes get this error: _"The User Account wasn't Found or is Protected"._
The Twitter account I use is not blocked and I can access the accounts that get this error via a browser with no problem, i.e. these accounts are public and they do not have my account blacklisted.

Do you have any idea why this happens? Is this some kind of a shadow ban by Twitter? 
Thank you

**Comment by mahrtayyab** [comment]
can you please share username of the s

**Comment by zherebilovgg** [comment]
![image](https://github.com/user-attachments/assets/4fe73f3f-2c93-4d97-abe8-2820ef550d41)
Here's a part of my logs where the method fails to scrape some accounts in a row.

**Comment by zherebilovgg** [comment]
UPD. I did more tests and noticed that there's a consistent set of accounts that the package is failing to scrape. I tried using different Twitter accounts for scraping, and none of them succeded. When I went to [tweeterid.com](https://tweeterid.com) to check if there was any problem with these accounts' user IDs, I noticed that it failed to convert to fetch the ID for them, while it works fine with the accounts that are successfully parsed by the package. (see the screenshot for example)
![image](https://github.com/user-attachments/assets/d7a9dc70-5454-4650-b22d-69c792a98881)
Can it be that the user IDs of these accounts are somehow hidden?
